RELEASE-NOTES.md

```diff
@@ -11,6 +14,7 @@
 **EUI-3155** Accessibility amends for non-descriptive headings
 **EUI-3550** Replace heading tags with spans on case edit pages
 
+
```

Comment:
_Nitpick:_
You might be okay in a `.md` file but lint complains about multiple blank lines of spacing... and this seems like an accidental addition.

Reply:
Addressed